docs/tutorial/snapcraft-guide.md

```diff
@@ -0,0 +1,181 @@
+# Snapcraft Guide (Ubuntu Software Center & More)
+
+This guide provides information on how to package your Electron application
+for the Ubuntu Software Center or any other Snapcraft environment.
+
+## Background and Requirements
+
+Together with the broader Linux community, Canonical aims to fix many of the
+common software installation problems with the [`snapcraft`](snapcraft.io)
+project. Snaps are containerized software packages that include required
+dependencies, auto-update, and work on all major Linux distributions without
+system modification.
+
+Snapcraft is the primary way to get your application into the Ubuntu Software
+Center, but the underlying [Snap Store](snapcraft-store) supports all major
+Linux distributions, too.
+
+There are two ways to create a `.snap` file:
```

Comment:
"There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors." 😉